kahypar/partition/context.h

```diff
@@ -657,6 +683,11 @@ static inline void sanityCheck(const Hypergraph& hypergraph, Context& context) {
       LOG << "Sum of individual part weights is less than sum of vertex weights";
       std::exit(-1);
     }
+    if (context.initial_partitioning.enable_early_restart || context.initial_partitioning.enable_late_restart) {
```

Comment:
How complicated would it be to get support for individual part weights?

Reply:
he is currently working on that and it will be integrated in separate PR

Reply:
Awesome! That's good news! 👍